operator/controller/src/test/java/io/apicurio/registry/operator/it/ITBase.java

```diff
@@ -62,7 +67,7 @@ public static void before() throws Exception {
                 .getOptionalValue(OPERATOR_DEPLOYMENT_PROP, OperatorDeployment.class)
                 .orElse(OperatorDeployment.local);
         deploymentTarget = ConfigProvider.getConfig().getOptionalValue(DEPLOYMENT_TARGET, String.class)
-                .orElse("k8s");
+                .orElse("kubernetes");
```

Comment:
double check if this is redundant or it was a bug

Reply:
It was a bug, we haven't discovered it because we didn't test on plain K8s.